libsolidity/interface/CompilerStack.cpp

```diff
@@ -841,3 +842,81 @@ string CompilerStack::computeSourceMapping(eth::AssemblyItems const& _items) con
 	}
 	return ret;
 }
+
+namespace {
+
+Json::Value gasToJson(GasEstimator::GasConsumption const& _gas)
+{
+    if (_gas.isInfinite || _gas.value > std::numeric_limits<Json::LargestUInt>::max())
+	return Json::Value(Json::nullValue);
```

Comment:
We can. I think `-1` is nicer because then we are not at the mercy of jsoncpp whether it serialises `null` or not. With `-1` it is clear the function is present, but we couldn't measure it.